src/main/antlr4/imports/mysql_idents.g4

```diff
@@ -12,7 +12,7 @@ user_token: (IDENT | QUOTED_IDENT | string_literal);
 
 name: ( id | tokens_available_for_names | INTEGER_LITERAL);
 id: ( IDENT | QUOTED_IDENT );
-literal: (float_literal | integer_literal | string_literal);
+literal: (float_literal | integer_literal | string_literal | NULL);
```

Comment:
does SET DEFAULT support the same values as DEFAULT? https://github.com/zendesk/maxwell/blob/928451d808d7313df92a335dc1c1738e43db3ab0/src/main/antlr4/imports/column_definitions.g4#L90